tests/acceptance/user-list-test.js

```diff
@@ -23,20 +23,25 @@ var page = PO.build({
     itemScope: 'tbody tr',
     item: {
       userName: PO.text('td', { index: 1 }),
-      role: PO.text('td', { index: 2})
+      role: PO.text('td', { index: 2}),
+      selectGender: PO.selectable('select'),
+      gender: PO.text('select option:selected')
```

Comment:
Using `PO.value` would return the value attribute from the option, and we want just the text since options are most likely used without the value attribute when using ember.